src/main/resources/static/css/typography.css

```diff
@@ -59,3 +59,25 @@ div.active a {
 a.link-large {
     font-size: large;
 }
+
+.custom-select-size {
+    font-size: 1.2rem;
+    text-align-last: center;
+}
+
+.custom-btn {
+    font-size: 25px;
```

Comment:
further down font-size is done by rem - so that seems to work. why do you need a custom button font size anyway?

Reply:
further down one is for error modal this one is for text in subject form modal

Reply:
yeah but you should still use rem